ui/package.json

```diff
@@ -44,7 +44,9 @@
     "postinstall": "patch-package",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "e2e:codegen": "concurrently \"npm run dev-no-ssl\" \"npx playwright codegen http://localhost:3000\""
+    "e2e:codegen": "concurrently \"npm run dev-no-ssl\" \"npx playwright codegen http://localhost:3000\"",
+    "cosmos": "cosmos",
+    "cosmos:export": "cosmos-export"
```

Comment:
Not sure how crucial this functionality is, but running the exported bundle didn't work for me. We should also `.gitignore` that generated directory if we're gonna keep this in

Reply:
Yeah, makes sense. I've removed for now, we can add back in and debug when we actually want to deploy.